File: platform/platform-api/src/com/intellij/ExtensionPoints.java
// Copyright 2000-2024 JetBrains s.r.o. and contributors. Use of this source code is governed by the Apache 2.0 license.
package com.intellij;

import com.intellij.openapi.diagnostic.ErrorReportSubmitter;
import com.intellij.openapi.extensions.ExtensionPointName;
import org.jetbrains.annotations.ApiStatus;

/** @deprecated don't add new constants to this class and use the replacement for the existing one */
@Deprecated(forRemoval = true)
public interface ExtensionPoints {
  /** @deprecated plugins should register their own {@link ErrorReportSubmitter} and/or use services */
  @ApiStatus.Internal
  @Deprecated(forRemoval = true)
  ExtensionPointName<ErrorReportSubmitter> ERROR_HANDLER_EP = ExtensionPointName.create("com.intellij.errorHandler");
}

File: platform/ml-impl/src/com/intellij/platform/ml/impl/tools/extensionPoints.kt
// Copyright 2000-2024 JetBrains s.r.o. and contributors. Use of this source code is governed by the Apache 2.0 license.
package com.intellij.platform.ml.impl.tools

import com.intellij.openapi.extensions.ExtensionPointName
import com.jetbrains.ml.FeatureProvider

internal val EP_NAME_FEATURE_PROVIDER: ExtensionPointName<FeatureProvider> = ExtensionPointName.create("com.intellij.platform.ml.featureProvider")
